opensrp-chw-core/src/main/java/org/smartregister/chw/core/dao/VisitDao.java

```diff
@@ -75,11 +75,10 @@ public static void undoChildVisitNotDone(String baseEntityID) {
     }
 
     public static boolean memberHasBirthCert(String baseEntityID) {
-        String sql = "select count(*) certificates " +
-                "from visit_details d " +
+        String sql = "select count(*) certificates from visit_details d " +
                 "inner join visits v on v.visit_id = d.visit_id COLLATE NOCASE " +
                 "where base_entity_id = '" + baseEntityID + "' COLLATE NOCASE and v.processed = 1 " +
-                "and (visit_key in ('birth_certificate','birth_cert') and details = 'GIVEN' or human_readable_details = 'Yes')";
+                "and (visit_key in ('birth_cert', 'birth_certificate') and human_readable_details = 'Yes')";
```

Comment:
The or clause was likely added because of another variant, you can use `(details = 'GIVEN' or human_readable_details = 'Yes')` instead